app/controllers/coronavirus_form/address_lookup_controller.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+class CoronavirusForm::AddressLookupController < ApplicationController
+  def submit
+    redirect_to check_your_answers_url if session[:check_answers_seen]
```

Comment:
It looks like this would prevent people from looking up a new address. Is that something we want to permit?

Reply:
Additionally, if someone goes back to this page, and re-enters a postcode, they get a 500 error, due to a DoubleRenderError (redirect_to and then a render controller_path called later).

Reply:
Now people can look up a new address but they have to go back to do it. They can either use the back links or browser buttons. Additionally the extra page we added seems to have fixed the 500 error